rebar.config.script

```diff
@@ -0,0 +1,13 @@
+NoCert = {d,'NO_CERTIFICATION',true},
+case os:getenv("NO_CERTIFICATION") of
+    false ->
+        CONFIG;
+    _ ->
+        case lists:keysearch(erl_opts, 1, CONFIG) of
+            {value, {erl_opts, Opts}} ->
+                NewOpts = [NoCert | Opts],
+                lists:keyreplace(erl_opts, 1, CONFIG, {erl_opts, NewOpts});
+            false ->
+                CONFIG ++ [{erl_opts, NoCert}]
```

Comment:
Is it necessary to append to the end? It is in general better to append at the front.